src/content/tutorials/observability/data-ingestion/logs/_index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Ingest logs into the Observability Platform
+linkTitle: Logs
+description: How to make new logs available in the Observability Platform in self-service.
+menu:
+  principal:
+    parent: tutorials-observability-data-ingestion
+    identifier: tutorials-observability-data-ingestion-logs
+weight: 50
+last_review_date: 2024-10-09
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-atlas
+user_questions:
+  - How to ingest new logs?
+  - Why do my clusters run Alloy?
+---
+
+By default, Giant Swarm clusters starting from release v29.0.0 are equipped with [Alloy](https://grafana.com/docs/alloy), it's a lightweight, high-performance log agent that collects system logs from your cluster and forwards them to a central [Loki](https://grafana.com/docs/loki) instance running on the management cluster.
+
+The Giant Swarm Observability Platform allows you to ingest logs from your workloads in a self-service way using labels to select which pods to ingest logs from.
+
+### Label your resources
+
+The label `giantswarm.io/logging` can be set to `true` or `false` on any Namespace or Pod. When the label isn't set it's considered `false`.
+
+- Pods in any namespace can be labeled
```

Comment:
I sée no notion that self service is only available starting from a cluster version when this would at least bé in 29.5.0 right ?

Reply:
It's mostl likely be v30, I added that